R/prep.R

```diff
@@ -30,46 +30,53 @@
     ts <- tracingState(FALSE)
 
     # eval
-    value <- tryCatch(
+    valueAndVisible <- try(
       {
-        capture.output( value <- eval(parse(text=expr), envir=env))
-        value
+        b <- parse(text=expr)
+        for(exp in b){
+          cl <- call('withVisible', exp)
+          capture.output(valueAndVisible <- eval(cl, envir=env))
+        }
+        valueAndVisible
       },
-      silent = TRUE,
-      error = function(e) '<ERROR>'
+      silent = getOption('vsc.trySilent', default=TRUE)
     )
+    if(class(valueAndVisible) == 'try-error'){
```

Comment:
The usage of `class(x) == 'try-error'` could be problematic when `x` has multi-element class. The robust way to do this is `inherits(x, 'try-error')` which detects if any class matches the `try-error`.

I notice there are numerous such usages in the codebase. It would be nice if all are fixed.